internal/fullnode/pod_builder.go

```diff
@@ -275,8 +275,6 @@ set -eu
 if [ ! -d "$CHAIN_HOME/data" ]; then
 	echo "Initializing chain..."
 	%s --home "$CHAIN_HOME"
-	# Remove because downstream containers check the presence of this file.
-	rm "$GENESIS_FILE"
```

Comment:
I'm not sure how this was working, because heighliner images do not have `rm`

Reply:
To clarify, in this comment, I meant that I'm not sure how it was working for existing deployments, because it did not work with the noble heighliner image. That was the reason for the change, otherwise we would not have needed it over the weekend while you were out.